eventdiscovery-sdk/src/main/res/layout/schedjoules_event_list.xml

```diff
@@ -12,9 +12,27 @@
                 android:layout_width="match_parent"
                 android:layout_height="match_parent"
                 app:layoutManager="LinearLayoutManager"
+                android:layout_marginBottom="20dp"
```

Comment:
I checked with layout inspector and this margin leaves a little bit too much space covered by the footer, can be changed to 24dp safely (I checked), to avoid incorrect proportions of the bottom items (no more events).